kompose.spec

```diff
@@ -190,11 +185,15 @@ export GOPATH=$(pwd):%{gopath}
 export GOPATH=$(pwd):$(pwd)/vendor:%{gopath}
 %endif
 
-#%gobuild -o bin/cli/main %{import_path}/cli/main
+# define buildflags here. these were taken from script/.build
+# we will need to periodically check these for consistentcy
+%define buildflags -tags experimental -ldflags="-w -X github.com/skippbox/kompose/version.GITCOMMIT=%{shortcommit}"
```

Comment:
I like all the macros be defined on the same place. Does it make sense to put them at the top of the spec file?